.github/workflows/default.yaml

```diff
@@ -38,6 +38,10 @@ jobs:
             platform: linux/amd64
             arch: amd64
             version: 7.4
+          -
+            platform: linux/amd64
+            arch: amd64
+            version: '8.0'
```

Comment:
@lmakarov for reference I needed to do this because when the version was being fed into the build it was treating `8.0` as `8` and causing issues.

Reply:
@sean-e-dietrich makes sense. Let's have all version numbers in quotes to keep them consistent.